devito/operator/profiling.py

```diff
@@ -52,8 +50,18 @@ def __init__(self, name):
         # Python-level timers
         self.py_timers = OrderedDict()
 
+        # Instance attributes
```

Comment:
remove this comment

Reply:
I'd better change/improve I think this comment